### PR TITLE
Chapter 6: Modelling Users

### PR DESCRIPTION
**Pull request by megatimranikram**

### CODE REVIEW

**Comment by megatimranikram** [comment]
ready

File: Gemfile.lock [hunk 1]
```diff
@@ -114,13 +114,21 @@ GEM
       dry-equalizer (~> 0.3)
       dry-inflector (~> 0.1, >= 0.1.2)
       dry-logic (~> 1.0, >= 1.0.2)
+<<<<<<< HEAD
```
Comment (ngocvt-0484) [inline Gemfile.lock]: please fix confict

File: config/application.rb [hunk 2]
```diff
@@ -10,4 +10,4 @@ class Application < Rails::Application
     config.i18n.available_locales = [:en, :ru]
     config.i18n.default_locale = :en
   end
-end
+end
```
Comment (ngocvt-0484) [inline config/application.rb]: add endline same as line 12
https://docs.google.com/document/d/104Csp4-vamVos5DEbi372nLBEfmqhj7h1ENYO8ebjCo/edit

**Comment by megatimranikram** [comment]
ready

File: Gemfile.lock (hunk 1 above)
Comment (megatimranikram) [reply Gemfile.lock]: done ms

File: config/application.rb (hunk 2 above)
Comment (megatimranikram) [reply config/application.rb]: done ms

**Comment by manhnt-0960** [comment]
commented
It's look like missing content

**Comment by megatimranikram** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**